.clang-tidy

```diff
@@ -6,6 +6,7 @@
 # b) too noisy (checks with > 100 new warnings are considered noisy, this includes e.g. cppcoreguidelines-*).
 
 HeaderFilterRegex: '^.*/(base|src|programs|utils)/.*(h|hpp)$'
+ExcludeHeaderFilterRegex: '^.*/contrib/.*\.(h|hpp)$'
```

Comment:
So will clang-tidy in CI complain about files in `contrib/` if l. 9 is not there? In theory, l. 8 excludes `contrib/` already.

Reply:
For some reason `HeaderFilterRegex` is not enough to stop `clang-tidy` from touching contrib headers ☹️